src/core/sync/mutex.d

```diff
@@ -92,7 +92,8 @@ class Mutex :
     this( Object o ) nothrow @trusted
     in
     {
-        assert( o.__monitor is null );
+        assert ( o, "Valid object instance muss be passed." );
```

Comment:
muss -> must

Or maybe, "Error: the argument passed to Mutex.this(Object o) is null!"